core/src/com/unciv/logic/city/CityConstructions.kt

```diff
@@ -159,7 +159,7 @@ class CityConstructions {
         val turnsToConstruction = turnsToConstruction(constructionName, useStoredProduction)
         val currentProgress = if (useStoredProduction) getWorkDone(constructionName) else 0
         if (currentProgress == 0) return "\n$cost${Fonts.production} $turnsToConstruction${Fonts.turn}"
-        else return "\n$currentProgress/$cost${Fonts.production}\n$turnsToConstruction${Fonts.turn}"
+        else return "\n$currentProgress/$cost${Fonts.production} $turnsToConstruction${Fonts.turn}"
```

Comment:
This is deliberate, since larger progresses (say, 800/1200) don't fit well on-screen with the turns to construction and turn numbers

Reply:
Could `Label.wrap` help? Result would be a little less predictable, however...

Reply:
Nah, looking at the screenies, lishaoxia moved the reason why something can't be queued to the bottom right pane so the numbers are all that's left after the name itself, and they should always fit in a line, at least that's the idea - and I do like what I see.

Reply:
The screenshot doesn't show how it looks when something is in progress, the problem was usually with XXX/XXXX progress.
Looks at the University in the queue, imagine it at 800/1200... and now imagine that it's in the middle of the queue and so there are up and down buttons...

Reply:
The line 161 of the code will Occupy 1 line, If we don't edit the line 162 and delete "\n", the original code will Occupy two lines.